source/direct_debit/index.html.md.erb

```diff
@@ -257,23 +256,31 @@ This would return payments 501 to 1000.
     </tr>
     <tr>
     <td nowrap><code>started</code></td>
-    <td>The user has visited the <code>next_url</code> and landed on the GOV.UK Pay page to enter their account information.</td>
+    <td>Your user has visited the <code>next_url</code> and landed on the GOV.UK Pay page to enter their account information.</td>
+    </tr>
+    <tr>
+    <td nowrap><code>abandoned</code></td>
+    <td>GOV.UK Pay did not set up the mandate, because your user cancelled or took longer than 90 minutes to finish entering their account information.</td>
+    </tr>
+    <tr>
+    <td nowrap><code>failed</code></td>
+    <td>GoCardless could not set up the mandate, for example because your user's account information was incorrect or your user's bank account does not support Direct Debit.</td>
     </tr>
     <tr>
     <td nowrap><code>pending</code></td>
-    <td>GOV.UK Pay has sent account details to the payment service provider (PSP), but the PSP has not yet confirmed if the mandate has been successfully set up.</td>
+    <td>GOV.UK Pay has sent your user's account information to GoCardless, but GoCardless has not yet confirmed if the mandate has been successfully set up.</td>
     </tr>
     <tr>
     <td nowrap><code>active</code></td>
-    <td>The PSP has confirmed that the mandate has been successfully set up.</td>
+    <td>GoCardless has confirmed that the mandate has been successfully set up.</td>
     </tr>    
     <tr>
     <td nowrap><code>cancelled</code></td>
-    <td>The user did not finish entering their bank account details, so the mandate was not set up.</td>
+    <td>The mandate was cancelled, for example because your user asked their bank to cancel the mandate.</td>
     </tr>   
     <tr>
     <td nowrap><code>inactive</code></td>
-    <td>You can no longer collect payments from the mandate, for example because the user cancelled it.</td>
+    <td>You can no longer collect payments because the mandate expired. A mandate usually expires if you do not collect payments against it for 12 months.</td>
```

Comment:
this should be 13 months, not 12 
https://www.directdebit.co.uk/FAQs/Pages/InactiveDirectDebits.aspx

Reply:
Ah thanks - fixed in https://github.com/alphagov/pay-tech-docs/pull/306/commits/7ad6f07c2dc6e261a7724b1f8e665c2180950b91.